<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="WebDriver" duration-ms="18018" started-at="2020-04-18T21:19:56 CEST" finished-at="2020-04-18T21:20:14 CEST">
    <groups>
    </groups>
    <test name="Test" duration-ms="18018" started-at="2020-04-18T21:19:56 CEST" finished-at="2020-04-18T21:20:14 CEST">
      <class name="TestNG.NewTest3">
        <test-method status="PASS" signature="launch()[pri:0, instance:TestNG.NewTest3@5b87ed94]" name="launch" is-config="true" duration-ms="8915" started-at="2020-04-18T21:19:56 CEST" finished-at="2020-04-18T21:20:05 CEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launch -->
        <test-method status="PASS" signature="verifytitle()[pri:0, instance:TestNG.NewTest3@5b87ed94]" name="verifytitle" duration-ms="77" started-at="2020-04-18T21:20:05 CEST" finished-at="2020-04-18T21:20:05 CEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifytitle -->
        <test-method status="FAIL" signature="verifyurl()[pri:0, instance:TestNG.NewTest3@5b87ed94]" name="verifyurl" duration-ms="80" started-at="2020-04-18T21:20:05 CEST" finished-at="2020-04-18T21:20:05 CEST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [www.edureka.com] but found [https://www.facebook.com/]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [www.edureka.com] but found [https://www.facebook.com/]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at TestNG.NewTest3.verifyurl(NewTest3.java:32)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyurl -->
        <test-method status="PASS" signature="close()[pri:0, instance:TestNG.NewTest3@5b87ed94]" name="close" is-config="true" duration-ms="293" started-at="2020-04-18T21:20:05 CEST" finished-at="2020-04-18T21:20:06 CEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- TestNG.NewTest3 -->
      <class name="TestNG.NewTest4">
        <test-method status="PASS" signature="verifytitle1()[pri:0, instance:TestNG.NewTest4@6e0e048a]" name="verifytitle1" duration-ms="8507" started-at="2020-04-18T21:20:06 CEST" finished-at="2020-04-18T21:20:14 CEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifytitle1 -->
      </class> <!-- TestNG.NewTest4 -->
    </test> <!-- Test -->
  </suite> <!-- WebDriver -->
</testng-results>
